appveyor.yml

```diff
@@ -20,10 +20,19 @@ init:
   - conda config --set always_yes yes
   # We need to do this first as other commands may not work with older versions of conda.
   - conda update conda
-  - conda install numpy scipy nose hdf5 --quiet
+  - conda install numpy scipy nose hdf5 sphinx mock --quiet
 
 install:
+  - echo on
+  #We need to run these two lines to have nmake available for doctests
+  - cd "C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC"
+  - vcvarsall amd64
+  #Return back to continue install of OpenMDAO
+  - cd C:\projects\openmdao-w047s
   - pip install .
 
 test_script:
   - nosetests
+  #after tests, need to cd down so that conf.py can be found for doctest
+  - cd openmdao/docs
+  - nmake /f Makefile.nmake doctest
```

Comment:
Here I call the doctest explicitly because our other method of setting the env var OPENMDAO_TEST_DOCS calls a linux-specific cmd